test/e2e/images_test.go

```diff
@@ -241,6 +241,17 @@ WORKDIR /test
 		Expect(result.OutputToStringArray()).Should(HaveLen(0), "dangling image output: %q", result.OutputToString())
 	})
 
+	It("podman pull by digest and list --all", func() {
+		session := podmanTest.Podman([]string{"pull", ALPINEAMD64DIGEST})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		// Prevent regressing on issue #7651.
+		result := podmanTest.Podman([]string{"images", "--all"})
+		result.WaitWithDefaultTimeout()
+		Expect(result).Should(Exit(0))
```

Comment:
You know my feelings about "run a test and only check exit code, and just blindly la-la-la about its actual output". I'm writing my own system test anyway, so this will be covered, but I do wish I could instill the habit of checking results.

Reply:
Thanks for the reminder, @edsantiago! It was not on purpose but happened in a hurry. Since it's not merged, I will improve the test.